### ISSUE TITLE
Flaky EventHubs Test (ServiceFabricProcessor.OptionsTests.RuntimeInformationTest())

### ISSUE THREAD

**Issue by chidozieononiwu** [Azure/azure-sdk-for-net#6056]
**Bug Description**
Microsoft.Azure.EventHubs.Tests.ServiceFabricProcessor.OptionsTests.RuntimeInformationTest fails occassionally on the windows agent.

***Exception or Stack Trace***
```
[xUnit.net 00:01:39.31]     Microsoft.Azure.EventHubs.Tests.ServiceFabricProcessor.OptionsTests.RuntimeInformationTest [FAIL]
Failed   Microsoft.Azure.EventHubs.Tests.ServiceFabricProcessor.OptionsTests.RuntimeInformationTest
Error Message:
 Assert.True() Failure
Expected: True
Actual:   False
Stack Trace:
   at Microsoft.Azure.EventHubs.Tests.ServiceFabricProcessor.OptionsTests.RuntimeInformationTest() in D:\a\1\s\sdk\eventhub\Microsoft.Azure.EventHubs\tests\ServiceFabricProcessor\OptionsTests.cs:line 191
18:02:09.3334175: End Microsoft.Azure.EventHubs.Tests.ServiceFabricProcessor.OptionsTests.UseInitialPositionProviderTest
Results File: D:\a\1\s\sdk\eventhub\Microsoft.Azure.EventHubs\tests\TestResults\VssAdministrator_fv-az379_2019-05-02_18_00_30.trx
Test Run Failed.

Total tests: 47. Passed: 43. Failed: 1. Skipped: 3.
Test execution time: 1.7163 Minutes
Test run for D:\a\1\s\artifacts\bin\Microsoft.Azure.EventHubs.Tests\Debug\net461\Microsoft.Azure.EventHubs.Tests.dll(.NETFramework,Version=v4.6.1)
Microsoft (R) Test Execution Command Line Tool Version 15.9.0
Copyright (c) Microsoft Corporation.  All rights reserved.
```

**To Reproduce**
Run CI build on the playground. Below are examples of where it occurs.
https://dev.azure.com/azure-sdk/playground/_build/results?buildId=25284&_a=summary
https://dev.azure.com/azure-sdk/public/_build/results?buildId=23739

**Expected behavior**
Occurs occassionally and randomly. Occurs only on the Windows Agent

**Setup (please complete the following information):**
 - OS: Windows

**Comment by weshaggard** [comment]
Another hit https://dev.azure.com/azure-sdk/public/_build/results?buildId=27029

@chidozieononiwu this keeps hitting randomly can you please submit a PR to disable this test and use this issue to track getting it fixed.

**Comment by JamesBirdsall** [comment]
Fixed by #7446
